crates/prelude/assets/extensions/gd.php

```diff
@@ -0,0 +1,697 @@
+const IMG_NEAREST_NEIGHBOUR = 16;
+
+/** @var int */
+const IMG_WEIGHTED4 = 21;
+
+/** @var int */
+const IMG_TRIANGLE = 20;
+
+/** @var int */
+const IMG_AFFINE_TRANSLATE = 0;
+
+/** @var int */
+const IMG_AFFINE_SCALE = 1;
+
+/** @var int */
+const IMG_AFFINE_ROTATE = 2;
+
+/** @var int */
+const IMG_AFFINE_SHEAR_HORIZONTAL = 3;
+
+/** @var int */
+const IMG_AFFINE_SHEAR_VERTICAL = 4;
+
+/** @var int */
+const GD_BUNDLED = 0;
+
+/** @var int */
+const IMG_FILTER_NEGATE = 0;
+
+/** @var int */
+const IMG_FILTER_GRAYSCALE = 1;
+
+/** @var int */
+const IMG_FILTER_BRIGHTNESS = 2;
+
+/** @var int */
+const IMG_FILTER_CONTRAST = 3;
+
+/** @var int */
+const IMG_FILTER_COLORIZE = 4;
+
+/** @var int */
+const IMG_FILTER_EDGEDETECT = 5;
+
+/** @var int */
+const IMG_FILTER_GAUSSIAN_BLUR = 7;
+
+/** @var int */
+const IMG_FILTER_SELECTIVE_BLUR = 8;
+
+/** @var int */
+const IMG_FILTER_EMBOSS = 6;
+
+/** @var int */
+const IMG_FILTER_MEAN_REMOVAL = 9;
+
+/** @var int */
+const IMG_FILTER_SMOOTH = 10;
+
+/** @var int */
+const IMG_FILTER_PIXELATE = 11;
+
+/** @var int */
+const IMG_FILTER_SCATTER = 12;
+
+/** @var string */
+const GD_VERSION = '2.3.3';
+
+/** @var int */
+const GD_MAJOR_VERSION = 2;
+
+/** @var int */
+const GD_MINOR_VERSION = 3;
+
+/** @var int */
+const GD_RELEASE_VERSION = 3;
+
+/** @var string */
+const GD_EXTRA_VERSION = '';
+
+/** @var int */
+const PNG_NO_FILTER = 0;
+
+/** @var int */
+const PNG_FILTER_NONE = 8;
+
+/** @var int */
+const PNG_FILTER_SUB = 16;
+
+/** @var int */
+const PNG_FILTER_UP = 32;
+
+/** @var int */
+const PNG_FILTER_AVG = 64;
+
+/** @var int */
+const PNG_FILTER_PAETH = 128;
+
+/** @var int */
+const PNG_ALL_FILTERS = 248;
+
+final class GdImage
+{
+}
+
+final class GdFont
+{
+}
+
+function gd_info(): array
+{
+}
+
+function imageloadfont(string $filename): \GdFont|false
+{
+}
+
+function imagesetstyle(\GdImage $image, array $style): bool
+{
+}
+
+function imagecreatetruecolor(int $width, int $height): \GdImage|false
+{
+}
+
+function imageistruecolor(\GdImage $image): bool
+{
+}
+
+function imagetruecolortopalette(\GdImage $image, bool $dither, int $num_colors): bool
+{
+}
+
+function imagepalettetotruecolor(\GdImage $image): bool
+{
+}
+
+function imagecolormatch(\GdImage $image1, \GdImage $image2): bool
+{
+}
+
+function imagesetthickness(\GdImage $image, int $thickness): bool
+{
+}
+
+function imagefilledellipse(\GdImage $image, int $center_x, int $center_y, int $width, int $height, int $color): bool
+{
+}
+
+function imagefilledarc(\GdImage $image, int $center_x, int $center_y, int $width, int $height, int $start_angle, int $end_angle, int $color, int $style): bool
+{
+}
+
+function imagealphablending(\GdImage $image, bool $enable): bool
+{
+}
+
+function imagesavealpha(\GdImage $image, bool $enable): bool
+{
+}
+
+function imagelayereffect(\GdImage $image, int $effect): bool
+{
+}
+
+function imagecolorallocatealpha(\GdImage $image, int $red, int $green, int $blue, int $alpha): int|false
```

Comment:
```suggestion
function imagecolorallocatealpha(GdImage $image, int $red, int $green, int $blue, int $alpha): int|false
```

same applies for all other functions.